frontend-student/src/components/courseDetails/CourseDetails.jsx

```
@@ -75,104 +73,643 @@ export default function CourseDetails() {
-                              className='bg-gray-200 py-2 px-4 rounded-md'
-                            >
-                              Video {index + 1}: {video.title}
+                  <div
+                    className={`p-4 flex justify-between items-center cursor-pointer transition-colors ${
+                      activeAccordion === milestone.id
+                        ? 'bg-blue-600 text-white'
+                        : 'bg-white text-gray-800 hover:bg-blue-50'
+                    }`}
+                    onClick={() => toggleMilestone(milestone.id)}
+                  >
+                    <div className='flex items-center'>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)